lib/sugar.js

```diff
@@ -0,0 +1,45 @@
+var SugarClient = require('sugar-client');
+var JSONAPIClient = require('json-api-client');
+var authClient = require('./auth');
+var config = require('./config');
+
+if (typeof window !== 'undefined') {
+  SugarClient.Primus = require('sugar-client/primus');
+  SugarClient.host = config.sugarHost;
+
+  var sugarApiClient = new JSONAPIClient(config.sugarHost, {
+    'Content-Type': 'application/json',
+    'Accept': 'application/json',
+  });
+
+  var sugarClient = new SugarClient();
+
+  auth.listen('change', function() {
```

Comment:
Switched all those refs to `auth`, I don't really consider that module a client.

The module should still emit change events, is it not doing that somewhere?